occurrences/models.py

```diff
@@ -286,33 +286,31 @@ def __str__(self):
         return f"{self.place_id}"
 
 
+class StudyLevel(TranslatableModel):
+    """
+    The Study Level is intended to be a hierarchical list of teaching degrees.
+    """
+
+    id = models.CharField(
+        max_length=255, primary_key=True
+    )  # PT-678 needs migration for 255 chars.
+    translations = TranslatedFields(
+        label=models.CharField(max_length=255, verbose_name=_("label"))
+    )  # Labels can have custom language translations.
+    level = models.PositiveIntegerField(
```

Comment:
Should the name of level field be a "sort" instead? The original implementation of StudyLevels was a hierarchical tuple of tuples and this field is here only to create that same hierarchy.

Reply:
Hmn, I don't think the original levels have hierarchical order (although the naming seems really so). They are just a flat list of study levels. I'm not sure if we'll have any use of it though.

Reply:
When they are shown in UI, they should have a predefined order. It does not look good if there are grades from 1 to 10 and they are in random order. In my opinion there should be some hierarchy and if the levels are stored in db, the hierarchy should be given by db.

Reply:
But if the hierarchy is only for UI, I would change the "level" to "sort"...

Reply:
Still thinking about this and changed my mind once again. "Sort" is a bad name because it's used by many query languages and ORMs. A "srt" or something would be better, but I think I just leave it "level", because we are really talking about study levels - not just any sortable instances, but levels of studying.